sycl/doc/syclcompat/README.md

```diff
@@ -106,9 +112,7 @@ inline dim3 operator-(const dim3 &a, const dim3 &b);
 } // syclcompat
 ```
 
-In SYCL, the fastest-moving dimension is the one with the highest index, e.g. in
-a SYCL 2D range iteration space, there are two dimensions, 0 and 1, and 1 will
-be the one that "moves faster". The compatibility headers for SYCL offer a
+The compatibility headers for SYCL offer a
```

Comment:
Does this line need format? It is too short.

Reply:
Yes, well spotted - thanks :+1: